### ISSUE TITLE
LMRTFY integration is broken

### ISSUE THREAD

**Issue by tposney** [tposney/midi-qol#706]
In GitLab by @nulmas on Feb 21, 2022, 14:12

I set up a new world with only midi-qol, DAE and LMRTFY enabled, and used the Full Automation setting to test it out. Even with that, players will just receive a chat message asking for a saving throw.

I'm using a fully updated Foundry and DnD 5e system.

**Comment by tposney** [comment]
It's not broken for me, so there must be a settings difference between the two setups.
Please export your midi-qol settings from the misc tab and attach it here - I can't look at the issue without them.

**Comment by tposney** [comment]
In GitLab by @nulmas on Feb 22, 2022, 24:13

Since midi-qol was updated a few hours ago, I decided to test it after sending the file and I figured out what the problem is. It's not on your end: when installing LMRTFY, Foundry will use version 2.0.0, which will fail to work with midi-qol. When updating, it will change to 1.2.9 which works properly. I should have checked, but since I installed the module literally before testing, I didn't even consider that could be the issue.

Sorry for the bother and thank you!
